Sources/BraintreeCore/BTJSON.swift

```diff
@@ -237,4 +237,20 @@ import Foundation
 
         return result
     }
+    
+    /// The `BTJSON` as a `BTPostalAddress`
+    /// - Returns: A `BTPostalAddress` parsed from the key/value pairs inside the `BTJSON`
+    public func asAddress() -> BTPostalAddress? {
```

Comment:
We will also be able to use this in the NativeCheckout module when that is refactored to use the Swift classes, which is why we opted to move it into `BTJSON` directly. Internal ticket for this is: DTBTSDK-2131